backend/app.mjs

```diff
@@ -10,8 +12,46 @@ app.use(express.urlencoded({ extended: true }));
 app.use(helmet());
 app.use(cors());
 
+const myBlockchain = new Blockchain();
```

Comment:
You can clean up this test code from the app.mjs file, (line 15 to 24), or move them to a separate file if you want to keep using them.